nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/launcher/connect/ConnectBroker.java

```diff
@@ -198,7 +198,7 @@ protected boolean isLocalPackageId(String pkgId) {
 
     protected boolean isRemotePackageId(String pkgId) {
         return PackageUtils.isValidPackageId(pkgId)
-                && NuxeoConnectClient.getPackageManager().findPackageById(pkgId) != null;
+                && NuxeoConnectClient.getPackageManager().getRemotePackage(pkgId) != null;
```

Comment:
Using `findRemotePackageById()` seems to be a more natural choice. But when you change the line from `getRemotePackage` to `findRemotePackageById`, and run the tests in `TestConnectBroker`, you'll find out that:

- When running all the tests together, sometimes the test `testDownloadUnknownPackage` fails randomly.
- When running the test `testDownloadUnknownPackage` individually, it always passed.